docs/source/install/common/datastore_crypto_key.rst

```diff
@@ -1,23 +1,22 @@
-The :doc:`Key-value store </datastore>` allows users to store encrypted values (secrets).
-These are stored using symmetric encryption (AES256). To generate a crypto key, run:
+The :doc:`Key-value store </datastore>` allows users to store encrypted values (secrets). These are
+stored using symmetric encryption (AES256). To generate a crypto key, run these commands:
 
-  .. code-block:: bash
+.. code-block:: bash
 
-    DATASTORE_ENCRYPTION_KEYS_DIRECTORY="/etc/st2/keys"
-    DATASTORE_ENCRYPTION_KEY_PATH="${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}/datastore_key.json"
+  DATASTORE_ENCRYPTION_KEYS_DIRECTORY="/etc/st2/keys"
+  DATASTORE_ENCRYPTION_KEY_PATH="${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}/datastore_key.json"
 
-    sudo mkdir -p ${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}
-    sudo st2-generate-symmetric-crypto-key --key-path ${DATASTORE_ENCRYPTION_KEY_PATH}
+  sudo mkdir -p ${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}
+  sudo st2-generate-symmetric-crypto-key --key-path ${DATASTORE_ENCRYPTION_KEY_PATH}
 
-    # Make sure only st2 user can read the file
-    sudo usermod -a -G st2 st2
-    sudo chgrp st2 ${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}
-    sudo chmod o-r ${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}
-    sudo chgrp st2 ${DATASTORE_ENCRYPTION_KEY_PATH}
-    sudo chmod o-r ${DATASTORE_ENCRYPTION_KEY_PATH}
+  # Make sure only st2 user can read the file
+  sudo usermod -a -G st2 st2
```

Comment:
For some unknown reason it's included here.

`st2` user is already part of `st2` group, added by [`deb`](https://github.com/StackStorm/st2-packages/blob/master/packages/st2/debian/preinst#L36) and [`rpm`](https://github.com/StackStorm/st2-packages/blob/master/packages/st2/rpm/preinst_script.spec#L21) packaging.

We can remove this line.

Reply:
Removed that line